include/c_bindings/fortran_array_view.hpp

```diff
@@ -78,7 +77,7 @@ namespace gridtools {
                 descriptor.rank = std::rank<Arr>::value;
                 descriptor.is_acc_present = false;
 
-                using indices = GT_META_CALL(meta::make_indices, std::rank<Arr>);
+                using indices = typename meta::make_indices<std::rank<Arr>>::type;
```

Comment:
I think you are aware of this because it will be removed anyway, but remember that this is not CUDA 8 compatible.

Reply:
Why not? Which bug is that? Isn't that the expansion of the macro? But yes, my next PR will replace it.